src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,128 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private Node<K,V>[] table;
+    private int size;
+    private int threshold;
```

Comment:
There is no need in this variable. It's used only once and you may just put (int) (table.length * DEFAULT_LOAD_FACTOR) there
```suggestion
```